### PR TITLE
Added script to call Monitor::RunFolder::Staging->move_to_analysis()

### PR DESCRIPTION
**Pull request by jenniferliddle**

### CODE REVIEW

File: bin/npg_move_runfolder [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+=head1 SYNOPSIS
+
+    C<npg_move_runfolder --runfolder_path path>
+
+=head1 DESCRIPTION
+
+This script moves 'incoming' to 'analysis' for a given runfolder,
+and changes the group name to that found in the staging configuration.
+
```
Comment (mgcam) [inline bin/npg_move_runfolder]: The description seems strange. Should read "This script moves a given run folder from 'incoming' directory to 'analysis' directory. "group name to that found in the staging configuration" is rather criptic.

File: bin/npg_move_runfolder [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+use Monitor::Staging;
+use Monitor::RunFolder::Staging;
+
+our $VERSION = '0';
+
+use Readonly;
+
+my $mon = Monitor::RunFolder::Staging->new_with_options(status_update => 0);
+my @ms = $mon->move_to_analysis();
```
Comment (mgcam) [inline bin/npg_move_runfolder]: It is still possible to overwrite status_update from the command line. Do we want this?

**Comment by mgcam** [comment]
The new script should be added to MANIFEST. The change should be added to Changes file. There should be a test for no status update

File: bin/npg_move_runfolder (hunk 2 above)
Comment (dkj) [reply bin/npg_move_runfolder]: I'm happy to keep that "free" functionality.

**Pull request merged**